fenix/app/src/main/java/org/mozilla/fenix/settings/sitepermissions/SitePermissionsFragment.kt

```diff
@@ -23,6 +24,11 @@ class SitePermissionsFragment : PreferenceFragmentCompat() {
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         setPreferencesFromResource(R.xml.site_permissions_preferences, rootKey)
+
+        val preferenceDescription = requirePreference<Preference>(R.string.pref_key_site_permissions_description)
+        if (!Config.channel.isMozillaOnline) {
```

Comment:
I think it'll be better if by default visibility is false and we check here to set to true.
```
if (Config.channel.isMozillaOnline) {
            preferenceDescription.isVisible = true
        }
```

Reply:
Okay, I'll update.